### PR TITLE
Mhofman/compartment make scope proxy

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

File: packages/ses/src/compartment-evaluate.js [hunk 1]
```diff
@@ -20,9 +20,10 @@ export const getCompartmentEvaluator = (compartmentFields, options) => {
   } = options;
 
   let safeEvaluate;
+  let scopeProxy;
 
   if (__moduleShimLexicals__ === undefined && !sloppyGlobalsMode) {
-    ({ safeEvaluate } = compartmentFields);
+    ({ safeEvaluate, scopeProxy } = compartmentFields);
```
Comment (kumavis) [inline packages/ses/src/compartment-evaluate.js:26]: `scopeProxy` on `compartmentFields` now?

Comment (mhofman) [reply packages/ses/src/compartment-evaluate.js:26]: Yeah #898 introduces a shared evaluator (saved on the `compartmentFields`), so in this change I simply save the `scopeProxy` that it uses.

**Comment by kumavis** [comment]
see https://github.com/endojs/endo/pull/881#pullrequestreview-759213278

**Comment by mhofman** [comment]
I've update this branch on top of the new #898 but I believe it should be abandoned in light of getting rid of scope proxies entirely (see #905)

**Pull request closed without merging**